app/assets/stylesheets/application.scss

```diff
@@ -1,12 +1,23 @@
 @import "foundation_and_overrides";
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,400italic);
 @import "colors";
 @import "top_bar";
 @import "welcome";
 @import "card";
 @import "report";
+@import "tabs";
 
 .full-width {
   margin: 0 auto;
   max-width: initial;
   width: 100%;
 }
+
+body {
+  background-image: image-url("landscape-mountains-nature-sky.jpg");
```

Comment:
Properties should be ordered background-attachment, background-image, background-position, background-repeat, background-size, min-height